azimuth/modules/dataset_analysis/syntax_tagging.py

```diff
@@ -42,56 +40,50 @@ def compute(self, batch: Dataset) -> List[TaggingResponse]:  # type: ignore
             tags: Newly calculated tags.
 
         """
-        spacy_pos = spacy.load(self.config.syntax.spacy_model)
+        syntax_options: SyntaxOptions = assert_not_none(self.config.syntax)
+        spacy_model = spacy.load(syntax_options.spacy_model)
 
         utterances = batch[self.config.columns.text_input]
-
-        # Always use the BERT tokenizer to ensure this feature works regardless of model
-        tokenizer = self.artifact_manager.get_tokenizer()
-        # truncate to max model size so it doesn't crash if the utterances are too long.
-        inputs = tokenizer(utterances, truncation=True)
-        batch_tokens = [tokenizer.convert_ids_to_tokens(tokens) for tokens in inputs["input_ids"]]
-        batch_tokens = [
-            [tok for tok in batch_token if tok not in [tokenizer.cls_token, tokenizer.sep_token]]
-            for batch_token in batch_tokens
-        ]
-
         records: List[TaggingResponse] = []
 
-        for tokens, utterance in zip(batch_tokens, utterances):
+        for utterance in utterances:
             tag: Dict[SmartTag, bool] = {
                 smart_tag: False
                 for family in [SmartTagFamily.extreme_length, SmartTagFamily.partial_syntax]
                 for smart_tag in SMART_TAGS_FAMILY_MAPPING[family]
             }
-            adds = {}
-            syntax_options: SyntaxOptions = assert_not_none(self.config.syntax)
 
-            do = self.spacy_pipeline(utterance)
-            adds[DatasetColumn.token_count] = len(tokens)
+            doc = spacy_model(clean_utterance(utterance))
+            # Remove punctuation for word count and smart tags
+            tokens = [token.text for token in doc if not token.is_punct]
 
-            if len(list(do.sents)) > 1:
-                tag[SmartTag.multi_sent] = True
+            # Some issues occur with other languages such as french if using doc.sents directly.
+            # Hence, we use an english sentencizer that seems to work better for similar languages.
```

Comment:
```suggestion
            # Hence, we use an English sentencizer that seems to work better for similar languages.
```
I know it's an uncapitalized anglais :)